pyqtorch/bitstrings.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+import numpy as np
+import torch
+
+
+def _basis_order(qs: tuple[int]) -> list[int]:
+    """
```

Comment:
Maybe a `torch.permute(mat, list_indices)` with `list_indices` corresponding to the index ordering of the original qubit support ?

Reply:
Hmm possibly something like that could work. I think it would for permuting the qubits on the wavefunction, but the matrices don't have the same dimensions. But maybe there is a smart way to reshape into that, permute, and then reshape back. I will think about it.

Reply:
Not solved yet, tracked now in https://github.com/pasqal-io/pyqtorch/issues/241